src/components/CampaignInteractionStepsForm.jsx

```diff
@@ -72,25 +34,9 @@ export default class CampaignInteractionStepsForm extends React.Component {
               answerActionsData: "",
               isDeleted: false
             }
-          ],
-      displayAllSteps: false
+          ]
     };
   }
-  /*
-    this.state.displayAllSteps is used to cause only the root interaction
-    node to render when the component first mounts.  ComponentDidMount sets
-    displayAllSteps to true, which forces render to run again, this time
-    including all interaction steps. This cuts half the time required to
-    render the interaction steps after clicking the header to expand the
-    interaction steps card.
-
-    FUTURE: reevaluate this after React >=16 upgrade
-  */
-  componentDidMount = () => {
```

Comment:
this wasn't working – I think it was supposed to have the effect I implemented [here](https://github.com/WorkingFamilies/Spoke/pull/33/files#diff-451422d2d4a57f669b164ae1f81fc7e99732a82bad7081e772916961b331746cR54)